index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <h1>Hello Microverse!</h1>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I suggest you use a different color from `#000 or Black` so you can check if your styling is working once you have added the styling link in the `index.html` file(using  `#000 or Black` color will not let you know if your styling is working or not as it is the default used color in HTML). I know you can do it :muscle: :muscle: